[2022-06-27 05:02:18,199] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-27T12:00:24.649090+00:00 [queued]>
[2022-06-27 05:02:18,260] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-27T12:00:24.649090+00:00 [queued]>
[2022-06-27 05:02:18,260] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 05:02:18,260] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 05:02:18,260] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 05:02:18,296] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): run_after_loop> on 2022-06-27 12:00:24.649090+00:00
[2022-06-27 05:02:18,299] {standard_task_runner.py:52} INFO - Started process 151481 to run task
[2022-06-27 05:02:18,325] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'run_after_loop', 'manual__2022-06-27T12:00:24.649090+00:00', '--job-id', '13', '--raw', '--subdir', '/home/ubuntu/airflow/airflow_env/lib/python3.8/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpe8e36eha', '--error-file', '/tmp/tmpxh33atxy']
[2022-06-27 05:02:18,327] {standard_task_runner.py:80} INFO - Job 13: Subtask run_after_loop
[2022-06-27 05:02:18,620] {task_command.py:370} INFO - Running <TaskInstance: example_bash_operator.run_after_loop manual__2022-06-27T12:00:24.649090+00:00 [running]> on host ubuntu
[2022-06-27 05:02:18,758] {taskinstance.py:1569} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=run_after_loop
AIRFLOW_CTX_EXECUTION_DATE=2022-06-27T12:00:24.649090+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-27T12:00:24.649090+00:00
[2022-06-27 05:02:18,760] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-27 05:02:18,770] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo 1']
[2022-06-27 05:02:18,798] {subprocess.py:85} INFO - Output:
[2022-06-27 05:02:18,798] {subprocess.py:92} INFO - 1
[2022-06-27 05:02:18,798] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-27 05:02:18,850] {taskinstance.py:1395} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=run_after_loop, execution_date=20220627T120024, start_date=20220627T120218, end_date=20220627T120218
[2022-06-27 05:02:18,882] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 05:02:18,966] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
